sources/packages/web/src/services/http/dto/Assessment.dto.ts

```diff
@@ -58,6 +58,7 @@ export interface AssessmentNOAAPIOutDTO {
   offeringStudyEndDate: string;
   msfaaNumber: string;
   disbursement: unknown;
+  eligibleAmount: number;
```

Comment:
I see that following fields are used in NOA form.io but not saved by the workflow. 

Are these fields not relevant to the part time application?

 
![image](https://github.com/bcgov/SIMS/assets/54600590/156afeaf-c0d9-4d5a-aad1-e3367bd96dca)

![image](https://github.com/bcgov/SIMS/assets/54600590/60fe3560-dbfd-4b96-9580-89d1955bf9da)

Reply:
Yes if you are talking about the form.io not saving the assessmentData by the PartTime application, yes there are fields that are used only for FullTime application.

Reply:
My question is, does part time noa require Living Allowance, Other allowable cost and Total Assessed Cost ? or they are disregarded part time NOA. 

I am doubtful that they may be needed for PT noa as well. But I may be wrong. 

But if they are needed, the value is never going to come for PT application unless I am missing something.

Reply:
I have confirmed with @HRAGANBC regarding this and there will be a future ticket that needs displaying of several other fields in the NOA screen. Will connect the ticket with out ticket for continuation.